RELEASES.md

```diff
@@ -39,44 +39,42 @@ be done from that branch. For example, once we release `v1.0.0`, a branch
 `release/1.0` will be created from that tag. All future patch releases will be
 done against that branch.
 
-### Pre-releases
+### Release Cadence
 
-Pre-releases, such as alphas, betas and release candidates will be conducted
-from their source branch. For major and minor releases, these releases will be
-done from main. For patch releases, these pre-releases should be done within
-the corresponding release branch.
+Minor releases are time based with a target cadence of 6 months. The next two
```

Comment:
Maybe this cadence should be marked tentative until 2026.
We have to have a couple of releases to evaluate whether this plan is feasible in practice

Reply:
I updated from "target cadence" to "initial cadence". I think the following sentence makes it clear that the cadence is open to change, let me know if that is not clear though